core/src/test/java/org/apache/calcite/test/JdbcTest.java

```diff
@@ -7397,6 +7444,38 @@ public MockDdlDriver() {
     public MyTable2[] mytable2 = { new MyTable2() };
   }
 
+  /**
+   * Schema for Date time test
+   */
+  public static class DateTimeSchema {
+    @Override public String toString() {
+      return "DateTimeSchema";
+    }
+
+    public final DateTimeData[] types = {
+        new DateTimeData(Time.valueOf("10:30:00"), new java.sql.Date(1560000000000L)),
+        new DateTimeData(Time.valueOf("12:00:00"), new java.sql.Date(1560002400000L)),
+        new DateTimeData()
```

Comment:
Can we add more records in this schema, especially some with null value columns.